src/mysql/types.cr

```diff
@@ -104,7 +104,17 @@ abstract struct MySql::Type
   decl_type Int24, 0x09u8
   decl_type Date, 0x0au8
   decl_type Time, 0x0bu8
-  decl_type DateTime, 0x0cu8
+  decl_type DateTime, 0x0cu8, ::Bytes do
```

Comment:
I just noticed `::Bytes` should be replaced by `::Time` here. check the `  decl_type String, 0xfeu8, ::String do...`